tests/integration/security/reachability/context.go

```diff
@@ -121,8 +121,8 @@ func (rc *Context) Run(testCases []TestCase) {
 		test.Run(func(ctx framework.TestContext) {
 			// Apply the policy.
 			policyYAML := file.AsStringOrFail(ctx, filepath.Join("../testdata", c.ConfigFile))
-			rc.g.ApplyConfigOrFail(rc.ctx, c.Namespace, policyYAML)
-			rc.ctx.WhenDone(func() error {
+			rc.g.ApplyConfigOrFail(ctx, c.Namespace, policyYAML)
```

Comment:
Just want to understand the code better myself a bit of more.

so the rc.ctx != the context is passed in to the test.Run(fn), right?

What's the need for rc.Context?

Reply:
Right. The one pass in is in the callback, so defined at run time.